### ISSUE TITLE
Error change Fragment

### ISSUE THREAD

**Issue by Takeshih6** [sjwall/MaterialTapTargetPrompt#137]
**Describe the bug**
Good
I miss an error at the moment that the TargetPrompt is visible within a fragment, and in that I make the change towards another fragment

**To Reproduce**
I handle an app with fragments

1. in the first case it is to open a fragment and that the prompt
2. then without closing the prompt, change the fragment
3. that's when the app closes, it can not be controlled with a try catch 
4. the new fragment does not show a prompt

**Stack trace**
   ```
 --------- beginning of crash
2018-09-28 00:06:07.261 21680-21680/com.app.client E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.app.client , PID: 21680
    java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.ViewParent android.view.View.getParent()' on a null object reference
        at uk.co.samuelwall.materialtaptargetprompt.SupportFragmentResourceFinder.getPromptParentView(SupportFragmentResourceFinder.java:62)
        at uk.co.samuelwall.materialtaptargetprompt.extras.PromptText.prepare(PromptText.java:124)
        at uk.co.samuelwall.materialtaptargetprompt.MaterialTapTargetPrompt.prepare(MaterialTapTargetPrompt.java:649)
        at uk.co.samuelwall.materialtaptargetprompt.MaterialTapTargetPrompt$3.onGlobalLayout(MaterialTapTargetPrompt.java:236)
        at android.view.ViewTreeObserver.dispatchOnGlobalLayout(ViewTreeObserver.java:945)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2540)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1633)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7786)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1004)
        at android.view.Choreographer.doCallbacks(Choreographer.java:816)
        at android.view.Choreographer.doFrame(Choreographer.java:751)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:990)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6863)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

```
```
   @NonNull
    @Override
    public ViewGroup getPromptParentView()
    {
        //noinspection ConstantConditions
        return (ViewGroup) this.fragment.getView().getParent();
    }

```

**Smartphone (please complete the following information):**
 - Device: Oneplus 6
 - OS: Android 9
 - Library version 2.12.4

**Comment by sjwall** [comment]
The error is occurring because the prompt is being called to show when the target fragment does not have a view. The obvious solution is to change the API to include null checks but this wouldn't solve the route cause.

At what point are you calling the show method? If it is from onCreateView then it needs to be wrapped with a call to Handler.post

```java
new android.os.Handler().post(new Runnable()
{
    @Override
    public void run()
    {
        // Create the prompt here
    }
})
```

**Comment by Takeshih6** [comment]
I call it on the onViewCreated, even change the call to leave after a POST query and still the problem. In both cases, try it with the solution you proposed to me. But the same error keeps coming.

**Comment by sjwall** [comment]
You should be able to fix the clipping by calling `setClipToView(null)`

Is the call to getFragment on a view class? My testing seemed to work although it is in a dialog fragment

**Comment by sjwall** [comment]
Thanks for the demo example, it helped massively to fix the error

It looks like for this scenario  `new MaterialTapTargetPrompt.Builder(getActivity()).setClipToView(null)` is what is needed

**Comment by sjwall** [comment]
Released in `v2.14.0` sorry for the delay
